src/uwtools/tests/utils/test_api.py

```diff
@@ -41,13 +41,6 @@ def test_ensure_data_source_stdin_ok():
     assert api.ensure_data_source(data_source=None, stdin_ok=True) is None
 
 
-def test_ensure_data_source_str_to_path():
-    val = "/some/path"
-    result = api.ensure_data_source(data_source=val, stdin_ok=False)
-    assert isinstance(result, Path)
-    assert result == Path(val)
```

Comment:
`_ensure_data_source()` no longer performs the `str` -> `Path` type conversion, so this test was obsolete.